apps/docs/src/.vitepress/components/DesignTokenHeader.vue

```diff
@@ -17,19 +18,19 @@ const emit = defineEmits<{
 </script>
 
 <template>
-  <div class="header">
+  <div class="header vp-raw">
```

Comment:
![image](https://github.com/SchwarzIT/onyx/assets/151019977/c32ab7c3-741d-458b-ae47-c76009673815)

should we center-align the headline with the button text or baseline it?